### ISSUE TITLE
Pangea django module is showing  ' "HttpRequest" not defined' error

### ISSUE THREAD

**Issue by annu12340** [pangeacyber/pangea-python#356]
After installing the pangea-django library, it is showing the following error
```   
class PangeaAuthMiddleware():
  File "C:\Users\admin\AppData\Local\Programs\Python\Python310\lib\site-packages\pangea_django\pangea_django_aungeaAuthMiddleware
    def __call__(self, request: HttpRequest):
NameError: name 'HttpRequest' is not defined
```

**Comment by annu12340** [comment]
The pangea-django code is my local doesn't have this line - `from django.http import HttpRequest.`
But it is present in the repo

Looks like pip package is not updated with the latest code

**Comment by kenany** [comment]
Published that fix in v1.0.1.

**Comment by doubletooth** [comment]
Hi @annu12340, can you try installing 1.0.1 and letting us know if you still run into the issue? Thanks!
